.pre-commit-config.yaml

```diff
@@ -33,15 +33,15 @@ repos:
         require_serial: true
 
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: debe478691c6711d19679b9973309e19368e79b6 # v1.92.1
+    rev: v1.96.2 # v1.92.1
     hooks:
       - id: terraform_validate
         args:
           - --hook-config=--retry-once-with-cleanup=true
       - id: terraform_fmt
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: 1ee2e387938ec90ec10a64f85fc0b4be527ade6a # v0.5.5
+    rev: v0.8.2 # v0.5.5
```

Comment:
The version comments in this file are now out of sync with the version specified, maybe needs removing